crates/polars-plan/src/dsl/function_expr/list.rs

```diff
@@ -257,6 +257,17 @@ pub(super) fn contains(args: &mut [Column]) -> PolarsResult<Option<Column>> {
     polars_ops::prelude::is_in(item.as_materialized_series(), list.as_materialized_series()).map(
         |mut ca| {
             ca.rename(list.name().clone());
+            if list.null_count() > 0 {
```

Comment:
This should happen in the `is_in` implementation

Reply:
Can you confirm the expected behavior for `is_in` null propagation? It appears that `is_in` does not currently propagate nulls at all:

```python
import polars as pl

df = pl.DataFrame(
    {
        "a": [1, 2, None, 3, None, None],
        "b": [[1], [1], [1], None, None, [None]],
    }
)

df.with_columns(
    pl.col("a").is_in("b").alias("a_in_b"),
)

# ┌──────┬───────────┬────────┐
# │ a    ┆ b         ┆ a_in_b │
# │ ---  ┆ ---       ┆ ---    │
# │ i64  ┆ list[i64] ┆ bool   │
# ╞══════╪═══════════╪════════╡
# │ 1    ┆ [1]       ┆ true   │
# │ 2    ┆ [1]       ┆ false  │
# │ null ┆ [1]       ┆ false  │ <- LHS is null
# │ 3    ┆ null      ┆ false  │ <- RHS is null
# │ null ┆ null      ┆ false  │ <- both sides are null
# │ null ┆ [null]    ┆ true   │ <- LHS is null, RHS contains null
# └──────┴───────────┴────────┘
```

Is that expected behavior?